content/docs/roadmap/v1.6.0.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "v1.6.0"
+weight = 1
++++
+
+# NSM v1.5.0
+
+## Release Schedule
+
+- **Feature Freeze** -  2022-09-05
+- **Release Candidate** -  2022-09-12
+- **Release Notes** -  2022-09-12
+- **Release** -  2022-09-19
+
+
+## Planned Enhancements
+
+
+### Update NSM sdk/api dependencies
+
+
+Planned to update next sdk/api dependencies:
+
+- Open Polic Agent
+- OpenTelemetry
+- gRPC
+- protobuf
+- JWT
+
+To keep NSM uptodate.
+
+
+### Improve stability and bug fixes
+
+Planned to resolve all reported issues with milestone `v1.6.0`.
+And and all new stability issues that will be reported before the release!
+
+
+### OPA integration for registries services
+
+Planned to add OPA policies for registries services.
+
+[See more details](https://github.com/networkservicemesh/sdk/issues/269) 
+
+### Add support for scaling k8s NSM registry
+
+Planned to improve NSM resiliency and availability via adding a possiblity to scale NSM k8s registry.
+
+[See more details](https://github.com/networkservicemesh/deployments-k8s/issues/6779)
+
+### Add cluster-info examples
+
+Planned to add an interdomain example on selecting endpoints via [k8s about api](https://github.com/kubernetes-sigs/about-api)
+
+### SR-IOV enablement on Equinix Metal n3 servers
+
+Equinix Metal is planning to make SR-IOV a default configuration on our [new n3 class of servers](https://feedback.equinixmetal.com/changelog/sr-iov-enabled-by-default-on-n3xlarge-servers).
+
+Planned to improve [integration-k8s-packet](https://github.com/networkservicemesh/integration-k8s-packet) to use new n3 class of servers.
+
+### Simplify deployments-k8s examples
+
+Planned to simplify NSM examples via getting rid of creating a dynamic files. 
+
+[See more details](https://github.com/networkservicemesh/deployments-k8s/issues/5436) 
+
+
+### Public cluster testing via clusterctl
+
+This makes our builds consistent and repeatable  across a wide variety of infrastructure environments. 
+
+Planned to migrate next repos to the `clusterctl`
+
+- integration-k8s-aws
+- integration-k8s-gke
+- integration-k8s-aks
```

Comment:
What about clusterctl and integration-k8s-packet?

Reply:
Oops!. Fixed.